db/migrate/1_drop_openstax_salesforce_users.rb

```diff
@@ -0,0 +1,14 @@
+# TODO: Remove after all servers have been migrated
+class DropOpenStaxSalesforceUsers < ActiveRecord::Migration[4.2]
+  def change
+    drop_table :openstax_salesforce_users do |t|
+      t.string :name
+      t.string :uid, null: false
+      t.string :oauth_token, null: false
+      t.string :refresh_token, null: false
+      t.string :instance_url, null: false
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
huh, I've never seen drop_table with columns, but I guess that makes it reversible

Reply:
yep :)